src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String text, String filePath) {
+        try (java.io.FileWriter writer = new java.io.FileWriter(filePath)) {
```

Comment:
According to the checklist, it's recommended to use try-with-resources to ensure that the resources are closed after the program is finished with them. However, the fully qualified name `java.io.FileWriter` is not necessary since you have already imported the class at the top. You can simplify this by just using `FileWriter`.